catkin_ws/src/motion_planning/scripts/transform_tester.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
Lets put this file in another folder somewhere called test, maybe in motion_planning